client/layout/guided-tours/tours/jetpack-site-accelerator-tour/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import meta from './meta';
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+export const JetpackSiteAcceleratorTour = makeTour(
+	<Tour { ...meta }>
+		<Step
+			name="init"
+			target=".jetpack-site-accelerator-settings .form-toggle__switch"
+			arrow="top-left"
+			placement="below"
+			style={ {
+				animationDelay: '0.7s',
+				zIndex: 1,
+			} }
+		>
+			{ ( { translate } ) => (
+				<Fragment>
+					<p>{ translate( 'Activate this toggle to enable Site Accelerator' ) }</p>
```

Comment:
Flip this toggle to enable Site Accelerator -- it optimizes your images and loads them from our super-powerful servers, taking the load off yours.

(Also feels like a place where we want to offer a little more detail/context.)